libs/core/src/components/pds-accordion/pds-accordion.scss

```diff
@@ -17,16 +18,38 @@ details {
   --spacing-summary-padding-inline-end: var(--pine-spacing-xxs);
 
   border-radius: var(--border-radius);
-  padding-inline: var(--spacing-details-padding-inline)
+  padding-inline: var(--spacing-details-padding-inline);
+
+  pds-icon {
+    transform: scaleY(1);
+    transition: transform var(--animation-transform-timing);
+  }
 }
 
 /* stylelint-disable-next-line */
 details[open] {
   background-color: var(--color-content-background);
+  transition: transform var(--animation-transform-timing);
 
   summary {
     color: var(--color-font-active);
     font-weight: var(--font-weight);
+
+    pds-icon {
+      transform: scaleY(-1);
+      transition: transform var(--animation-transform-timing);
+    }
+  }
+
+  /* stylelint-disable-next-line selector-no-qualifying-type */
+  .pds-accordion__body {
+    transition: max-height 1s ease-in-out;
+
+    /* stylelint-disable-next-line selector-no-qualifying-type */
+    &.open {
+      max-height: 5000px;
```

Comment:
How do you feel about using `max-content` here instead of 5k?

Reply:
@anechol see commit [2e0f1e5](https://github.com/Kajabi/pine/pull/154/commits/2e0f1e546fc490de2e13fdfabc3aafe38df65dc6) for change